ors-api/src/test/java/org/heigit/ors/api/servlet/listeners/ORSInitContextListenerTest.java

```diff
@@ -16,16 +16,16 @@ void setUp(){
     }
     @ParameterizedTest
     @CsvSource(value = {
-            "https://example.com/test-repo/planet,      https://example.com, test-repo, planet",
-            "https://example.com/test-repo/planet/,     https://example.com, test-repo, planet",
-            "' https://example.com/test-repo/planet ',  https://example.com, test-repo, planet",
-            "' https://example.com/test-repo/planet/ ', https://example.com, test-repo, planet",
-            "' http://example.com/test-repo/planet/ ',  http://example.com, test-repo, planet"
+            "https://example.com/foo1/foo2/foo3/test-repo,      https://example.com, test-repo",
```

Comment:
Please add this test and change the code to make it pass:
```suggestion
            "https://test-repo.openrouteservice.org/service/rest/ors-graphs-fastisochrones/, https://test-repo.openrouteservice.org/service/rest, ors-graphs-fastisochrones",
```

Reply:
Sure. I'll need to add it myself. GH can't

Reply:
Added. But i left it as example.com